coalib/results/SourceRange.py

```diff
@@ -126,6 +126,40 @@ def __json__(self, use_relpath=False):
             _dict['file'] = relpath(_dict['file'])
         return _dict
 
+    def __str__(self):
+        """
+        Creates a string representation of the SourceRange object
+        Here are doctest examples of the three different representations
```

Comment:
I would leave this sentence away, the user can see that examples are following :)